lib/registry.rb

```diff
@@ -0,0 +1,51 @@
+require 'date'
+require './spec/spec_helper.rb'
+require_relative 'errors.rb'
+
+ROOMS = (1..20).to_a
+module Hotel
+  class Registry
+    attr_accessor :reservations, :okay_rooms
+
+    def initialize
+      @reservations = []
+      @okay_rooms = []
```

Comment:
`@okay_rooms` probably doesn't need to be an instance variable.  You could make it a local variable in `available?` which could return a list of rooms which are available.  

Making it an instance variable which several methods are playing with and making it an accessor opens you up to side effects.